modules/reference/pages/cluster-properties.adoc

```diff
@@ -1030,6 +934,7 @@ If `null`, the property is disabled, and traffic is not limited.
 
 ---
 
+[[kafka_throughput_limit_node_out_bps]]
```

Comment:
in what units (we should say below bytes per second for added clarity)

Reply:
> in what units (we should say below bytes per second for added clarity)

Yes, bytes per second